examples/simple/posix/UB-PTHREAD8-good.c

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2015 Runtime Verification, Inc. (RV-Match team). All Rights Reserved.
+
+#include<stdio.h>
+#include<string.h>
+#include<pthread.h>
+#include<stdlib.h>
+
+
+
+pthread_t thread_id;
+pthread_mutex_t lock;
+
+void * locker_thread(void * ptr) 
+{
+    //trylock should succeed as there is no other thread with the lock acquired.
+    pthread_mutex_trylock(&lock); 
+    pthread_mutex_unlock(&lock);
+    return NULL;
+}
+
+int main(void)
+{
+    int err = 0;
+    if (pthread_mutex_init(&lock, NULL) != 0)
+    {
+        printf("\n mutex init failed\n");
+        return 1;
+    }
+
+
+   
+    err = pthread_create(&thread_id, NULL, &locker_thread, NULL);
+    if (err != 0) 
+    {
+        printf("Thread spawning error");
+        return 1;
+    }
+
+   
+    
+    pthread_join(thread_id, NULL);
+    pthread_mutex_destroy(&lock);
+    
+    printf("Successful Execution\n");
+    
+    return 0;
+}
```

Comment:
The -good one seems a lot larger and different than the -bad one.  Can they get closer to each other?  I think people will like to see quickly the fix.